js/packages/web/src/hooks/useArt.ts

```diff
@@ -136,7 +137,7 @@ export const useArt = (id?: PublicKey | string) => {
   const { metadata, editions, masterEditions, whitelistedCreatorsByCreator } =
     useMeta();
 
-  const key = typeof id === 'string' ? id : id?.toBase58() || '';
+  const key = pubkeyToString(id);
```

Comment:
We were just planning on replacing all the toStrings with .equals, which compares arrays and is probably cheaper. Why not do that?